chapters/annotations.tex

```diff
@@ -888,6 +888,11 @@ \subsection{Component Instance}\label{component-instance}
 Sub-connectors in a hierarchical connector are only shown when they can be connected to.
 Non-connector components are only shown in the diagram layer.
 
+\subsubsection{Default Outline for Missing Graphics}\label{default-outline-for-missing-graphics}
+If the icon of a class does not contain any graphical primitives (including inherited ones, and regardless of visibility; but excluding connectors) it is said to be missing graphics.
```

Comment:
I would like a clarification of _regardless of visibility_.  Does it mean that you won't get an automatic icon even if all graphic primitives have `visible = false`?  I think that would be a useful rule, as it makes it possible to turn off the automatic icon if one only wants to see the connectors.

Reply:
Yes. Can add that in the non-normative section.